#!/usr/bin/env python
PACKAGE = "selfie_perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_range", double_t, 0, "max_range", 1.0, 0.5, 2.0)
gen.add("min_range", double_t, 0, "min_range", 0.03, 0.01, 0.05)
gen.add("min_segment_size", double_t, 0, "min_segment_size", 0.04, 0.01, 0.06)
gen.add("max_segment_size", double_t, 0, "max_segment_size", 0.5, 0.1, 1.0)
gen.add("segment_threshold", double_t, 0, "segment_threshold", 0.03, 0.01, 0.05)
gen.add("min_to_divide", double_t, 0, "min_to_divide", 0.03, 0.01, 0.05)

exit(gen.generate(PACKAGE, "detect_obstacles", "DetectObstacles"))
